### PR TITLE
Fix for Blank Case Claim Args

### PR DESCRIPTION
**Pull request by ctsims**
Allow case claim to skip blank arguments in request. Needed for apps with multiple case claim fields, otherwise all fields are mandatory for search.

Fix for: 
https://manage.dimagi.com/default.asp?253026

### CODE REVIEW

**Review by wpride — APPROVED** [review]

**Comment by wpride** [comment]
CS getting in on the JS action!

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
             var fields = $(".query-field");
             var model = this.parentModel;
             fields.each(function(index) {
-                payload[model[index].get('id')] = this.value;
+                if ( !(this.value === "") ) {
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js]: haha you're so java. `this.value !== ''`

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js (hunk 1 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js]: look at those ridiculous double quotes!

**Comment by proteusvacuum** [comment]
👍 
Thanks! I had thought that the server side did this already, but alas.

**Pull request merged**